### DISCUSSION TITLE
how to resolve out of memory issue?

### DISCUSSION THREAD

**Discussion by jeromedoss** [cypress-io/cypress#28991] [General]
![image](https://github.com/cypress-io/cypress/assets/139545420/9a6a0629-5ebb-4e81-a70c-2520e1cd1ed6)

**Comment by jeromedoss** [comment]
@jennifer-shehane

**Comment by samelawrence** [comment]
Have you tried enabling the [experimental](https://docs.cypress.io/guides/references/experiments) memory management feature? You can enable it in your `cypress.config.ts` file.

```
export default defineConfig({
  e2e: {
    experimentalMemoryManagement: true
  }
}
```

**Reply by jeromedoss** [reply]
yes have tried

**Reply by krisodb** [reply]
Put also numTestsKeptInMemory: 0
And env var CYPRESS_NO_COMMAND_LOG=1
And in our CI I've split the batch in multiple smaller sub sets. Loosing time spinning up docker each time, but less memory issue's.
Check https://github.com/cypress-io/cypress/discussions/25557

**Comment by abhishek104** [comment]
I tried all the above things mentioned above but still I am getting same error. I am using Cypress 13.6.6 and tried with all the latest version of Chrome, Edge and Electron. Can anyone urgently help please?
I have e2e Regression Test Suite and my PC Ram is 64 GBs.

**Comment by rima90** [comment]
Hello, 
any solution about this issue please?
